typing/printtyp.ml

```diff
@@ -2522,6 +2521,3 @@ let tree_of_path = tree_of_path Other
 let tree_of_modtype = tree_of_modtype ~ellipsis:false
 let type_expansion mode ppf ty_exp =
   type_expansion ppf (trees_of_type_expansion mode ty_exp)
-let tree_of_type_declaration ident td rs =
-  with_hidden_items [{hide=true; ident}]
-    (fun () -> tree_of_type_declaration ident td rs)
```

Comment:
I'm not sure why this change is included here, and don't know if it is correct. Is it intentional?

Reply:
No that was a leftover from my tests to check if it was really necessary to also hide the identifier in this case. This has been fixed.